programs/mango-v4/src/instructions/group_edit.rs

```diff
@@ -12,6 +13,10 @@ pub fn group_edit(
     testing_opt: Option<u8>,
     version_opt: Option<u8>,
     deposit_limit_quote_opt: Option<u64>,
+    fees_pay_with_mngo_opt: Option<bool>,
+    fees_mngo_bonus_factor_opt: Option<f32>,
+    fees_swap_mango_account_opt: Option<Pubkey>,
```

Comment:
let's have a consistent name for this, I've seen:
- buyback_fees
- discount_buyback_fees
- fees_pay_with_mngo
- fees_swap

I'd go for `buyback_fees` I think

Reply:
all done, please recheck though